pkg/device/device-util.go

```diff
@@ -486,27 +564,30 @@ func GetDiskDetails() ([]apis.Device, error) {
 		return nil, err
 	}
 	for _, diskIter := range diskList {
-		metaName, err := getDiskMetaName(diskIter.DiskName)
+
+		metaName, err := getDiskMetaName(diskIter.DiskPath)
 		if err != nil {
-			klog.Errorf("Device LocalPV: getDiskMetaName Failed %s", diskIter.DiskName)
+			klog.Errorf("Device LocalPV: getDiskMetaName Failed %s", diskIter.DiskPath)
 			continue
 		}
-		id, err := getDiskIdentifier(diskIter.DiskName)
+		id, err := getDiskIdentifier(diskIter.DiskPath)
 		if err != nil {
-			klog.Errorf("Device LocalPV: getDiskIdentifier Failed %s", diskIter.DiskName)
+			klog.Errorf("Device LocalPV: getDiskIdentifier Failed %s", diskIter.DiskPath)
 			continue
 		}
-		free, err := GetFreeCapacity(diskIter.DiskName)
+		free, err := GetFreeCapacity(diskIter.DiskPath)
 		if err != nil {
-			klog.Errorf("Device LocalPV: GetFreeCapacity Failed %s", diskIter.DiskName)
+			klog.Errorf("Device LocalPV: GetFreeCapacity Failed %s", diskIter.DiskPath)
 			continue
 		}
-		result = append(result, apis.Device{metaName, id,
-			*resource.NewQuantity(int64(diskIter.Size), resource.DecimalSI),
-			*resource.NewQuantity(int64(free*1024*1024), resource.DecimalSI)})
+		result = append(result, apis.Device{
+			Name: metaName,
+			UUID: id,
+			Size: *resource.NewQuantity(int64(diskIter.Size), resource.BinarySI),
```

Comment:
@praveengt Changed this to `BinarySI` since that is the correct unit that we should use.